src/events/rawMessageReactionRemove.ts

```diff
@@ -11,7 +11,7 @@ export default class extends Event {
 
 	public run(data: WSMessageReactionRemove) {
 		const channel = this.client.channels.get(data.channel_id) as TextChannel;
-		if (!channel || !channel.readable) return;
+		if (!channel || !channel.readable || channel.type !== 'text') return;
```

Comment:
This should check for `'dm'`, so `news` aren't excluded:
```suggestion
		if (!channel || !channel.readable || channel.type === 'dm') return;
```

Reply:
Ditto everywhere else.